posthog/plugins/test/plugin_archives.py

```diff
@@ -22,6 +22,11 @@
     "UEsDBAoAAAAAAARCYlIAAAAAAAAAAAAAAAA6AAkAaGVsbG93b3JsZHBsdWdpbi0yZWQzNGE3NTNmYTAyNTRjMGMzMDFmMTc1NWQ0YjE0NTZiMjEwZGMyL1VUBQAByGQ+YFBLAwQKAAAAAAAEQmJSXDzpciMAAAAjAAAARAAJAGhlbGxvd29ybGRwbHVnaW4tMmVkMzRhNzUzZmEwMjU0YzBjMzAxZjE3NTVkNGIxNDU2YjIxMGRjMi8uZ2l0aWdub3JlVVQFAAHIZD5gLmlkZWEKbm9kZV9tb2R1bGVzCnlhcm4tZXJyb3IubG9nCgpQSwMECgAAAAgABEJiUqd95gdAAQAAsQIAAEIACQBoZWxsb3dvcmxkcGx1Z2luLTJlZDM0YTc1M2ZhMDI1NGMwYzMwMWYxNzU1ZDRiMTQ1NmIyMTBkYzIvaW5kZXguanNVVAUAAchkPmB9Uk1PwzAMvedXmAmpLVQdH0fEDZC4IXFECKWd10XKkipxKNO0/46btGNj0nKIEvv5+Tkv0m9MA8tgGlLWgEcK3ZsOrTL5FhprlqotodW2lhp2BWwF8OK4txorbdt89o5EyrQQOqAVQheLL2bFCTKxpXhirGK7J2sQHoFcQLETQh4r6pxt0PvnbzSU47CX8KeskQ33PBJGvAdD6JhT9lJRAlUtUp6NqayEm1FgzPnj3ERwDbeFiDC1hNS8Yj0dOlLop6bD+p/7yFaotc0+WUTWW6cX2RlsLV1Epqkqvp4BX046U0U8n4FrVX+ZsK7HAr6+jE+b36cn2KURHXvhTGIYfFjbRWDn8KezjjyXpmkPfkgZA4cGlUPhfA6vgyjDX4bbOek2ezs91KhtL8Te3wNBkCeh08OOilIQruDugel/AVBLAwQKAAAACAAEQmJSoG8NBdsAAABeAQAARQAJAGhlbGxvd29ybGRwbHVnaW4tMmVkMzRhNzUzZmEwMjU0YzBjMzAxZjE3NTVkNGIxNDU2YjIxMGRjMi9wbHVnaW4uanNvblVUBQAByGQ+YEWPPU8DMQyG9/sV5haWqrd3QWKAigmJoQPqkF58iSGXHI6jUqr+d/IhuMVSnvd5Y/naAfRezdjvoN+jcwEOgZ3uNyVI7Aq3IkvcDYMhsem0HcM8vIYo+2AGWyrn0lhcMuRbT2McmRah4Ev/mREFxGL7G5TX8BQCKHhUvIGXN0BNRb5r9VlR7ZHX+L39iI2OwU9kMn/PL4BrnZl/4qXIJ8XVq+zvooNVch+BfN2elYfVkctSnShM3qw84sgoORFO+E81Tio5aZt+VpvxKxGjzsGkXMTKb3keu1v3C1BLAQIAAAoAAAAAAARCYlIAAAAAAAAAAAAAAAA6AAkAAAAAAAAAEAAAAAAAAABoZWxsb3dvcmxkcGx1Z2luLTJlZDM0YTc1M2ZhMDI1NGMwYzMwMWYxNzU1ZDRiMTQ1NmIyMTBkYzIvVVQFAAHIZD5gUEsBAgAACgAAAAAABEJiUlw86XIjAAAAIwAAAEQACQAAAAAAAQAAAAAAYQAAAGhlbGxvd29ybGRwbHVnaW4tMmVkMzRhNzUzZmEwMjU0YzBjMzAxZjE3NTVkNGIxNDU2YjIxMGRjMi8uZ2l0aWdub3JlVVQFAAHIZD5gUEsBAgAACgAAAAgABEJiUqd95gdAAQAAsQIAAEIACQAAAAAAAQAAAAAA7wAAAGhlbGxvd29ybGRwbHVnaW4tMmVkMzRhNzUzZmEwMjU0YzBjMzAxZjE3NTVkNGIxNDU2YjIxMGRjMi9pbmRleC5qc1VUBQAByGQ+YFBLAQIAAAoAAAAIAARCYlKgbw0F2wAAAF4BAABFAAkAAAAAAAEAAAAAAJgCAABoZWxsb3dvcmxkcGx1Z2luLTJlZDM0YTc1M2ZhMDI1NGMwYzMwMWYxNzU1ZDRiMTQ1NmIyMTBkYzIvcGx1Z2luLmpzb25VVAUAAchkPmBQSwUGAAAAAAQABADhAQAA3wMAACgAMmVkMzRhNzUzZmEwMjU0YzBjMzAxZjE3NTVkNGIxNDU2YjIxMGRjMg==",
 )
 
+HELLO_WORLD_PLUGIN_GITHUB_SUBDIR_ZIP = (
```

Comment:
lol I hate these tests - something to consider in the future

Reply:
I know 😆 Trying to edit this file was a battle. Think these tests are still valuable as I found a couple errors in my implementation when running them, but there's probably a more efficient way of at least storing the test data.